build.sh

```diff
@@ -1,37 +1,77 @@
-set -e 
+#!/bin/sh
+# Build script of python package
+
+set -e
 name="oval-graph"
 module="oval_graph"
 
+unit_test() {
+    python3 -m pytest
+}
+
+commit_version() {
+    version=$1
+    module=$2
+    git add ${module}/__init__.py
+    git add oval-graph.spec
+    git commit -m "${version}"
+    git tag "${version}"
+    git push --follow-tags
+    git push upstream master
```

Comment:
My suggestion (i.e. not merge condition) would be to perform those things manually, maybe have the script print those commands, so it is easy to execute them.
~You want to be sure that you didn't forget to commit a file, or that unwanted files are not committed, run `git status` before committing etc.~ I see that you already do that, that's cool.